script/bootstrap

```diff
@@ -65,7 +65,7 @@ install_packages_apt()
     if [ "$PLATFORM" = "Raspbian" ]; then
         sudo apt-get --no-install-recommends install -y binutils-arm-none-eabi gcc-arm-none-eabi gdb-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
     elif [ "$PLATFORM" = "Ubuntu" ]; then
-        sudo apt-get --no-install-recommends install -y ca-certificates wget
+        sudo apt-get --no-install-recommends install -y ca-certificates wget gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
```

Comment:
Do you mind avoiding this for now and adding this into the "Toolchain Setup" section of the `README.md` for the platforms that require it?